src/patternfly/components/Switch/switch.hbs

```diff
@@ -0,0 +1,14 @@
+<label for="{{id}}" class="pf-c-switch{{#if disabled}} pf-m-disabled{{/if}}" role="switch" aria-label="..."  aria-checked="{{#if off}}false{{else}}true{{/if}}">
+  <input id="{{id}}" type="checkbox" class="pf-c-switch__input"{{#if disabled}} aria-disabled="true" disabled{{/if}} />
```

Comment:
Since the `<input>` element supports `disabled` natively, we can remove `aria-disabled` here.